core/src/main/scala/org/apache/spark/rpc/netty/Dispatcher.scala

```diff
@@ -33,7 +33,7 @@ import org.apache.spark.util.ThreadUtils
 /**
  * A message dispatcher, responsible for routing RPC messages to the appropriate endpoint(s).
  */
-private[netty] class Dispatcher(nettyEnv: NettyRpcEnv) extends Logging {
+private[netty] class Dispatcher(nettyEnv: NettyRpcEnv, numUsableCores: Int) extends Logging {
```

Comment:
Should we document the behavior when `numUsableCores` is set to 0 in the comment above?